examples/trace/grpc/hello_world_client.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import grpc
+
+import hello_world_pb2
+import hello_world_pb2_grpc
+
+from opencensus.trace.tracer import Tracer
+from opencensus.trace.exporters import stackdriver_exporter
+from opencensus.trace.ext.grpc import client_interceptor
+
+HOST_PORT = 'localhost:50051'
```

Comment:
You might look into the 'portpicker' package on PyPI for picking unused ports instead of hardcoding.  Hardcoding can run into problems when you're running tests on CI machines and can have collisions which lead to hard-to-debug failures.

Reply:
... but you can do this later.

Reply:
Good suggestion! This is just an example and will not be running on CI. I'll do that when I added the system test for grpc integration.

Reply:
You're right, for a sample app, hard-coding ports is probably the right thing to, because it's simpler for the person trying to learn from the code.